build/docker/Dockerfile

```diff
@@ -41,15 +41,16 @@ ENV GITHUB_CLI_VERSION 1.14.0
 RUN curl --silent --location "https://github.com/cli/cli/releases/download/v${GITHUB_CLI_VERSION}/gh_${GITHUB_CLI_VERSION}_linux_amd64.tar.gz" \
     | tar xvz -C /out/usr/local/bin --strip-components=2 gh_${GITHUB_CLI_VERSION}_linux_amd64/bin/gh && gh --version
 
-# Remaining dependencies are controlled by go.mod
 WORKDIR /src
-ENV CGO_ENABLED=0 GOPROXY=https://proxy.golang.org,direct
 
-# The authenticator is a runtime dependency, so it needs to be in /out
-ENV AWS_IAM_AUTH_VERSION 0.5.2
+# The authenticator is a runtime dependency.
```

Comment:
I had to re-order this to trigger a completely new build and a new image id. I definitely want to redo the whole building image thing by adding a version tag which will trigger a new build of a new layer.

Reply:
Changing `ENV AWS_IAM_AUTH_VERSION 0.5.7` alone would result in the layers below it to be rebuilt. Reordering the instructions shouldn't be required.

Reply:
It was the layers above this.

Reply:
Removed. :)